### PR TITLE
Tools: sim_vehicle.py: flake8 compliance

### PR DESCRIPTION
**Pull request by peterbarker**
The vast majority of this is white-space changes.

```
pbarker@bluebottle:~/rc/ardupilot(sv-flake8)$ flake8 Tools/autotest/sim_vehicle.py
pbarker@bluebottle:~/rc/ardupilot(sv-flake8)$ 
```

### CODE REVIEW

File: Tools/autotest/sim_vehicle.py [hunk 1]
```diff
@@ -528,8 +561,9 @@ def start_mavproxy(opts, stuff):
         if stuff["sitl-port"]:
             cmd.extend(["--sitl", simout_port])
 
-    # If running inside of a vagrant guest, then we probably want to forward our mavlink out to the containing host OS
-    ports = [p + 10 * cmd_opts.instance for p in [14550,14551]]
+    # If running inside of a vagrant guest, then we probably want to
+    # forward our mavlink out to the containing host OS
```
Comment (OXINARF) [inline Tools/autotest/sim_vehicle.py]: Since you are modifying this, don't want to put another commit to move this down? It doesn't apply to the statement below it, but two lines below.

File: Tools/autotest/sim_vehicle.py [hunk 2]
```diff
@@ -549,7 +586,7 @@ def start_mavproxy(opts, stuff):
         extra_cmd += " " + stuff["extra_mavlink_cmds"]
 
     if opts.mavproxy_args:
-        cmd.extend(opts.mavproxy_args.split(" "))  # this could be a lot better..
+        cmd.extend(opts.mavproxy_args.split(" "))  # this could be a lot better
```
Comment (OXINARF) [inline Tools/autotest/sim_vehicle.py]: Humm, so this comment doesn't need to be above the line like others do?

File: Tools/autotest/sim_vehicle.py [hunk 3]
```diff
@@ -766,23 +935,29 @@ def generate_frame_help():
...
+                  location)
 
 if cmd_opts.delay_start:
     progress("Sleeping for %f seconds" % (cmd_opts.delay_start,))
     time.sleep(float(cmd_opts.delay_start))
 
 try:
     if cmd_opts.no_mavproxy:
-        time.sleep(3)  # Just wait to output the last command after run_in_terminal_window.sh
+        time.sleep(3)  # output our message after run_in_terminal_window.sh's
```
Comment (OXINARF) [inline Tools/autotest/sim_vehicle.py]: Again, the comment doesn't need to be in it's own line?

File: Tools/autotest/sim_vehicle.py [hunk 4]
```diff
@@ -87,17 +95,23 @@ def format_option(self, option):
...
+        optparse.OptionParser.__init__(self,
+                                       *args,
+                                       formatter=formatter,
+                                       **kwargs)
 
     def error(self, error):
-        """Override default error handler called by optparse.OptionParser.parse_args when a parse error occurs; raise a detailed exception which can be caught"""
+        """Override default error handler called by
+optparse.OptionParser.parse_args when a parse error occurs; raise a
+detailed exception which can be caught"""
```
[the remark this answers is not in the document]
Comment (peterbarker) [reply Tools/autotest/sim_vehicle.py]: Will fix.  This is `emacs`'s default way to format it.

File: Tools/autotest/sim_vehicle.py [hunk 5]
```diff
@@ -157,19 +173,23 @@ def kill_tasks_cygwin(victims):
     """Shell out to ps -ea to find processes to kill"""
     for victim in list(victims):
         pids = cygwin_pidof(victim)
-#        progress("pids for (%s): %s" % (victim,",".join([ str(p) for p in pids])))
+#        progress("pids for (%s): %s" % (victim,",".join([ str(p) for
+#        p in pids])))
```
[the remark this answers is not in the document]
Comment (peterbarker) [reply Tools/autotest/sim_vehicle.py]: Yes, I believe so.

I'll split the line up to be more legible.

File: Tools/autotest/sim_vehicle.py [hunk 6]
```diff
@@ -191,7 +211,7 @@ class BobException(Exception):
 
 
 def kill_tasks():
-    """Clean up stray processes by name.  This is a somewhat shotgun approach"""
+    """Clean up stray processes by name.  This is a the shotgun approach"""
```
[the remark this answers is not in the document]
Comment (peterbarker) [reply Tools/autotest/sim_vehicle.py]: Apparently I took a somewhat shotgun approach to English....

File: Tools/autotest/sim_vehicle.py (hunk 1 above)
Comment (peterbarker) [reply Tools/autotest/sim_vehicle.py]: Will do.

File: Tools/autotest/sim_vehicle.py (hunk 2 above)
Comment (peterbarker) [reply Tools/autotest/sim_vehicle.py]: Yep.

File: Tools/autotest/sim_vehicle.py (hunk 3 above)
Comment (peterbarker) [reply Tools/autotest/sim_vehicle.py]: If it wasn't annoying flake8 I didn't necessarily play with stuff.

In this case I think the comment belongs on the line - it's apologising for dodgily using `time.sleep(3)`

**Comment by peterbarker** [comment]
I've pushed this in - thanks @OXINARF !
